ui/app/AppLayouts/Chat/components/PopupMenu.qml

```diff
@@ -0,0 +1,75 @@
+import QtQuick 2.12
```

Comment:
I see you put the file in `Chat/components`. It's fine if we only plan to use it in Chat, but I assume we'll use it elsewhere too, so it might be better in `shared/` then.